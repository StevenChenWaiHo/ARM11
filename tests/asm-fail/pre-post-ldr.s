str r1,[r0,#8],[r0,#40]
