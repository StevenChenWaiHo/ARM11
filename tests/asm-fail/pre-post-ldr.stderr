tests/asm-fail/pre-post-ldr.s:1:16: Already seen extra pre index arguments, cannot use post index arguments
